rmake.py

```diff
@@ -100,7 +100,8 @@ def config_cmd():
         rocm_path = os.getenv( 'ROCM_CMAKE_PATH', "C:/github/rocm-cmake-master/share/rocm")
         cmake_executable = "cmake"
         #set CPACK_PACKAGING_INSTALL_PREFIX= defined as blank as it is appended to end of path for archive creation
-        cmake_platform_opts.append( f"-DCPACK_PACKAGING_INSTALL_PREFIX={rocm_path}" )
+        cmake_platform_opts.append( f"-DCPACK_PACKAGING_INSTALL_PREFIX=" )
+        cmake_platform_opts.append( f"-DCMAKE_INSTALL_PREFIX=\"C:/hipSDK\"" )
```

Comment:
It's purely a matter of style, I would suggest using `'` quotes when the string contains `"` and use `"` when the string contains a `'` to minimize the need for backslash escapes. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#string-quotes).

This file is probably still undergoing rapid changes across in multiple repositories, so it's probably a waste of time to make those sorts of tiny improvements. But, I figured I should mention it anyway, for when the time comes.